packages/styles/src/tile.scss

```diff
@@ -767,15 +765,30 @@ $fd-line-tile-badge-horizontal-spacing: 0.375rem !default;
     width: auto;
     padding-block: 0;
     padding-inline: $fd-line-tile-horizontal-padding;
-    box-shadow: none;
-    background: transparent;
+    box-shadow: var(--fdTile_Box_Shadow, var(--sapContent_Shadow0));
+    background: var(--fdTile_Background, var(--sapTile_Background)); 
+    border: var(--fdTile_Border, var(--sapTile_BorderColor));
```

Comment:
border is a combination of border-width, border-style and border-color. Something like:
`border: 1px solid red`. `--sapTile_BorderColor` specifies just the color of the border. Please check , but should should something like:
```
border: 0.0625rem solid var(--fdTile_Border, var(--sapTile_BorderColor));
```